### DISCUSSION TITLE
Can I remove the highlight path on url when I delete the search content

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5257] [Q&A]
![image](https://user-images.githubusercontent.com/62207095/227705776-14fa38fc-3c5b-4e99-a1af-b0c0b2841132.png)

Can I just stop the highlight from happening after the search bar is null?

**Comment by squidfunk** [comment]
Currently not, but search highlighting and result navigation will soon receive an update, see https://github.com/squidfunk/mkdocs-material/issues/4125#issuecomment-1374882924

**Reply by SPyofgame** [reply]
Oh ok 👍 btw is the text get parsed by highlighting is a part of intended ?

**Reply by squidfunk** [reply]
I'm sorry, but I don't understand the question.

**Reply by SPyofgame** [reply]
I remember that a few days ago when I search for something

It shows the content like this: 
**Highl** ight  <--- a space in between
Instead of:
**Highl**ight

**Reply by SPyofgame** [reply]
![image](https://user-images.githubusercontent.com/62207095/227714110-d86b869c-e999-45ed-8212-900199466e92.png)
like that instead of this
![image](https://user-images.githubusercontent.com/62207095/227714145-d07d6298-29b9-4a0b-b562-f2984a1dae4f.png)

**Reply by squidfunk** [reply]
No, there should be no spaces. I'm not observing this on our own documentation, e.g. when searching for [`"docs"`](https://squidfunk.github.io/mkdocs-material/getting-started/?h=docs):

<img width="751" alt="Bildschirm­foto 2023-03-25 um 12 18 36" src="https://user-images.githubusercontent.com/932156/227714338-cad3a7b9-43e0-4aaf-bb37-647c8e320136.png">

**Reply by facelessuser** [reply]
It's possible it is an unintended side effect in my own custom styling. I would not use my documentation as a reference as to whether Material is working as my custom styling may inadvertently introduce some unintended behavior.

**Reply by SPyofgame** [reply]
Oh sorry, I have a mistake in referencing that.

**Reply by facelessuser** [reply]
Well, if anything, I now know I need to fix some styling rules for search in my docs 🙃.
